keyboards/converter/ps2_usb/led.c

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2011 Jun Wako <wakojun@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "stdint.h"
+#include "ps2.h"
+#include "led.h"
+
+
+void led_set(uint8_t usb_led)
```

Comment:
Travis is complaining about the usage of `IS_LED_ON()` here, although I can see host.h is definitely included *before* led.h. It might be picking up the wrong led.h (there is one in tmk_core/common which defines the `USB_LED_*` constants).

I think it would be better to move this function into ps2_usb.c as `led_set_kb()` and delete this file.